pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/KeyValueSchemaTest.java

```diff
@@ -234,6 +234,58 @@ public void testNotAllowNullSchemaEncodeAndDecode() {
         assertEquals(bar, barBack);
     }
 
+    @Test
+    public void testkeyValueEncodingTypeSchemaEncodeAndDecode() {
```

Comment:
Similar comments as in `MessageImplTest.java `. I don't see how encoding type was changed in KeyValueSchema.